### ISSUE TITLE
Accuracy on CONLL2003 dataset

### ISSUE THREAD

**Issue by SandeepNaidu** [explosion/spaCy#2169]
Is spaCy 2.0 trained on CONLL2003 dataset? As per the documentation it is trained on OntoNotes 5.0 which is better and good. But when I run the evaulation on CONLL 2003 dataset I see the following results. The F-Measure is in single digits for NER. Is that because the model is not trained on CONLL dataset? I thought a model trained on OntoNotes 5.0 should do good on CONLL 2003 also. What am I missing here?

python -m spacy evaluate en_core_web_lg eng.train.json

    Results

    Time               52.05 s
    Words              203621
    Words/s            3912
    TOK                100.00
    POS                87.99
    UAS                0.00
    LAS                0.00
    NER P              6.24
    NER R              9.46
    NER F              7.52

$ python -m spacy evaluate en_core_web_sm eng.train.json

    Results

    Time               47.34 s
    Words              203621
    Words/s            4302
    TOK                100.00
    POS                87.71
    UAS                0.00
    LAS                0.00
    NER P              5.36
    NER R              8.06
    NER F              6.43

**Comment by honnibal** [comment]
Well the annotation scheme is entirely different. Like, spaCy's model outputs PERSON, CoNLL 2003 expects PER --- so even if you get the entity right with spaCy, that will be counted as a mistake. spaCy also outputs a lot of entities CoNLL doesn't expect, e.g. DATE, CARDINAL, etc. Those will also all be counted as mistakes. CoNLL expects MISC, which spaCy never outputs. etc etc etc etc -- even smaller details of the annotation schemes in the corpora will matter a lot.

It almost never makes sense to evaluate a model trained on one corpus on the test set of another corpus. You can train spaCy on the CoNLL 2003 dataset by using the `python -m spacy convert` command. From memory we get around 92% if pre-trained vectors are used.
